pkg/validation/spec/info.go

```diff
@@ -212,11 +200,7 @@ func (i *Info) UnmarshalNextJSON(opts jsonv2.UnmarshalOptions, dec *jsonv2.Decod
 	if err := opts.UnmarshalNext(dec, &x); err != nil {
 		return err
 	}
-	x.Extensions.sanitize()
-	if len(x.Extensions) == 0 {
-		x.Extensions = nil
-	}
-	i.VendorExtensible.Extensions = x.Extensions
+	i.Extensions = internal.SanitizeExtensions(x.Extensions)
```

Comment:
did this intentionally change from assigning to `i.VendorExtensible.Extensions`?

Reply:
answering my own question, it resolves the same and is not ambiguous